### DISCUSSION TITLE
PRO Features

### DISCUSSION THREAD

**Discussion by ghdragunoff** [remotely-save/remotely-save#879] [Q&A]
How do I use the pro features? They became unavailable after October 31st. I used to sync obsidian with pcloud via a solution that was added by @fyears

**Comment by fyears** [comment]
please refresh the expiration date in remotely save settings. the free beta period is extended now.

**Reply by ghdragunoff** [reply]
I couldn't connect. In the remotely save profile, after logging in and enabling the functions, the date 2024-11-01T00:00:00.000Z. is displayed in the PRO table. Also inside Obsidian, the plugin does not see PRO functions after connection 🫠
![IMG_0833](https://github.com/user-attachments/assets/d4c9d99a-36d0-4c2b-a5ed-73ed5771dd78)

**Comment by heatfighter** [comment]
Hmm, at least for me it does not seem to work. When I signed in, all feature expired at "2025-01-01T00:00:00.000Z", but when I clicked disable and then enable again, the expiration date was set to "2024-11-01T00:00:00.000Z":
![image](https://github.com/user-attachments/assets/44e7745b-ced3-48fd-9c4a-db25d3a21346)
Here, I disabled smart_conflict and yandex_disk. The other ones I did not touch. Funny is also the text "The feature is FREE while beta before October 31, 2024. You can enable it right now." even though we are already in November 2024...

**Reply by maxirass** [reply]
Same problem here. I tried to enable google_drive, but there is no button to buy the pro feature and the expired at is set to 2024-11-01

**Reply by am4c130d** [reply]
Same - enabled the feature at remotelysave.com, expiration date is 2024-11-01. The feature doesn't appear in the config page.

**Reply by ghdragunoff** [reply]
@fyears 🙏🏽

**Reply by hansimglueck** [reply]
I tried to enable it for the first time and also got the expired expiration date... whats the plan?

**Reply by jade2020** [reply]
Same issue here...Any updates?
